src/map-interactions.react.js

```diff
@@ -187,36 +195,39 @@ var MapInteractions = React.createClass({
       mouseWheelTimeout: timeout,
       mouseWheelLastValue: lastValue
     });
-  },
+  }
   /* eslint-enable complexity, max-statements */
 
-  _zoom: function _zoom(delta, pos) {
+  _zoom(delta, pos) {
 
     // Scale by sigmoid of scroll wheel delta.
-    var scale = 2 / (1 + Math.exp(-Math.abs(delta / 100)));
+    let scale = 2 / (1 + Math.exp(-Math.abs(delta / 100)));
     if (delta < 0 && scale !== 0) {
       scale = 1 / scale;
     }
-    this.props.onZoom({pos: pos, scale: scale});
+    this.props.onZoom({pos, scale});
     window.clearTimeout(this._zoomEndTimeout);
     this._zoomEndTimeout = window.setTimeout(function _setTimeout() {
```

Comment:
> the closure can be () => this.props.onZoomEnd()

I've had problems when using window.setTimeout with fat arrow functions in the past, and since they are asynchronous they can be a little tricky to test. So I intentionally left these out for now. Easy enough to update them when everything else is confirmed to be working.